lib/rules/camelcase.js

```diff
@@ -161,7 +161,7 @@ module.exports = {
             switch (parent.type) {
                 case "Property":
                     return (
-                        parent.parent.type === "ObjectPattern" &&
+                        (parent.parent.type === "ObjectPattern" || parent.parent.type === "ObjectExpression") &&
```

Comment:
This change looks good. Can you also update JSDoc description for this function since this doesn't fall under destructuring?

Reply:
Updated 👍🏻